addons/account/migrations/9.0.1.1/noupdate_changes.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version='1.0' encoding='utf-8'?>
+<openerp>
+  <data>
+    <record id="email_template_edi_invoice" model="mail.template"><field name="body_html">
```

Comment:
Ouch, as you haven't backported #1045 and you are using v9 script, `CDATA` tags have been stripped... You need to take care of this.

Reply:
SHIT

Reply:
I included now that improvement in the other PR.

Reply:
OK, waiting for Travis for merging.

Reply:
And doing now the update of this PR

Reply:
Now `CDATA` tags are included 😌